tests/basic/tst_basic.cpp

```diff
@@ -330,7 +330,7 @@ void tst_Basic::testQNamespace()
     QCOMPARE(AnotherTU::staticMetaObject.enumerator(0).name(), "ATTestEnum");
     QCOMPARE(QMetaEnum::fromType<AnotherTU::ATTestEnum>().name(), "ATTestEnum");
 
-#ifdef __cpp_inline_variables
+#if defined (__cpp_inline_variables) && Q_CC_MSVC > 192000000
```

Comment:
why this change?

Reply:
Simply because VS2017 cannot cope with the `inline variables`-thing you need, but defines `__cpp_inline_variables`.